hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMultiTargetMessageTask.java

```diff
@@ -74,6 +75,17 @@ protected CompletableFuture<Object> processInternal() {
         return finalResult;
     }
 
+    protected void rethrowInReduce(Map<Member, Object> map, boolean ignoreMemberLeftException) throws Throwable {
```

Comment:
Tentative style comment: I think it would be better to add this method to `ExceptionUtil` as a static method. For example:
```java
/**
 * If there's any Throwable instance in the coll, throw it.
 */
public static void rethrowFromCollection(Collection<?> coll, Class<Throwable> ... ignored)
```

This way it will be more readable when you see it in code:
```java
    protected Object reduce(Map<Member, Object> map) throws Throwable {
        rethrowFromCollection(map.values(), ignored: MissingMemberException.class);
```
The `ignored:` label might be displayed by IDE.

When you see `rethrowInReduce(map, true);`, it's not as clear, you have to check out the method.

Reply:
:+1: done